src/Components/Organisms/FOCardWithImage/index.tsx

```diff
@@ -28,18 +27,25 @@ export const FOCardWithImage = memo((props: IFOCardWithImage) => {
       {...materialUIProps}
       sx={{
         ...sx,
-        border: "1px solid #D4D4D4",
+        borderWidth: "1px",
+        borderStyle: "solid",
+        borderColor: "grey.300",
       }}
     >
       <CardContent>
         {option && (
           <FAFormControlLabel
             label={optionLabel || "optionLabel"}
-            testID="ControlLabel-test"
+            testID="form-control-label"
             control={option}
+            sx={{ mb: 3 }}
           ></FAFormControlLabel>
         )}
-        {svg}
+        <Box display="flex" justifyContent="center" width="100%">
+          <Box display="flex" mb={3} sx={{ width: "250px", height: "180px" }}>
```

Comment:
I think it just needs one `<Box>` component. remove box in line 45. Add `mx="auto"` and `maxWidth="250px"` props in line 44 to centralize the child component.

Reply:
updated @gilangdarmawan